datastore/load_test.go

```diff
@@ -439,6 +439,36 @@ type withUntypedInterface struct {
 	Field interface{}
 }
 
+func TestLoadCivilTimeInNonUTCZone(t *testing.T) {
+	src := &pb.Entity{
+		Key: keyToProto(testKey0),
+		Properties: map[string]*pb.Value{
+			"Time": {ValueType: &pb.Value_TimestampValue{TimestampValue: &timestamppb.Timestamp{Seconds: 1605504600}}},
+		},
+	}
+	dst := &struct{ Time civil.Time }{
+		Time: civil.Time{},
+	}
+	want := &struct{ Time civil.Time }{
+		Time: civil.Time{
+			Hour:   5,
+			Minute: 30,
+		},
+	}
+	loc, _ := time.LoadLocation("Africa/Cairo")
```

Comment:
Catch and fail on errors from LoadLocation as well (and in L468)

Reply:
Done